.github/workflows/pr-wasm-preview.yml

```diff
@@ -0,0 +1,210 @@
+name: PR WASM Preview Deployment
+
+on:
+  pull_request:
+    branches:
+      - main
+      - dev
+    types:
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - 'mm2src/**'
+      - Cargo.lock
+      - Cargo.toml
+      - rust-toolchain.toml
```

Comment:
`.cargo` should also be included. If it doesn't costs us, we can just include the whole repo for safety (e.g., adding new directory and forgetting to add it here).